src/backend/distributed/operations/shard_split.c

```diff
@@ -375,13 +392,85 @@ SplitShard(SplitMode splitMode,
 							   SplitOperationName[splitOperation],
 							   SplitTargetName[splitOperation])));
 	}
-	else if (PlacementMovedUsingLogicalReplicationInTX)
+}
+
+
+/*
+ * ErrorIfMultipleNonblockingSplitInTheSameTransaction will error if we detect multiple
+ * nonblocking split operation in the same transaction.
+ */
+static void
+ErrorIfMultipleNonblockingSplitInTheSameTransaction(SplitOperation splitOperation)
+{
+	if (PlacementMovedUsingLogicalReplicationInTX)
 	{
 		ereport(ERROR, (errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
 						errmsg("multiple shard movements/splits via logical "
 							   "replication in the same transaction is currently "
-							   "not supported")));
+							   "not supported"),
+						errhint("If you wish to execute multiple shard movements/splits"
```

Comment:
hint is only relevant in the split/isolate/move case, confusing in the create_distributed_table_concurrently case